packages/coinstac-simulator/test/mocks/exec-computation.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
```

Comment:
Hard to tell whether this totally covers us, but `CommandComputation`s in both `local` and `remote` should be good. Also, this test is _noisy_.